ligo/src/token/fa2.mligo

```diff
@@ -74,9 +74,9 @@ let transfer_item (store, ti : storage * transfer_item): storage =
   in List.fold transfer_one ti.txs store
```

Comment:
In light of using always the `store` name, I think maybe we should consider renaming the `acc` in `transfer_item` too?

This seems to work just as fine: 
```ocaml
let transfer_item (store, ti : storage * transfer_item): storage =
  let transfer_one (store, tx : storage * transfer_destination): storage =
    let valid_from_ = check_sender (ti.from_, store) in
    if tx.amount = 0n then store
    else (
      let valid_token_id =
        if tx.token_id = unfrozen_token_id then tx.token_id
        else if tx.token_id = frozen_token_id then (
          if sender = store.admin then tx.token_id
          else (failwith ("FROZEN_TOKEN_NOT_TRANSFERABLE"): token_id)
        ) else (failwith ("FA2_TOKEN_UNDEFINED") : token_id)
      in credit_to
        ( tx.amount, tx.to_, valid_token_id
        , debit_from (tx.amount, valid_from_, valid_token_id, store)
        )
    )
  in List.fold transfer_one ti.txs store
```

here we loose the ability to access the outer-most `store`, but in this case it makes no difference (logically).

Checking the resulting Michelson code there is indeed a slight change for the better (because ti doesn't have to fetch `store.admin` from the further away), so I suggest that we either:
- replace `acc` with `store` as in the block above
- replace `store.admin` with `acc.admin`